cmd/homi/setup/flags.go

```diff
@@ -320,121 +345,137 @@ var (
-		Usage: "governance proposer policy (0: RoundRobin, 1: Sticky, 2: WeightedRandom) [default: 0]",
-		Value: params.DefaultProposerPolicy,
+		Name:    "ist-proposer-policy",
+		Usage:   "governance proposer policy (0: RoundRobin, 1: Sticky, 2: WeightedRandom) [default: 0]",
+		Aliases: []string{"genesis.consensus.istanbul.policy"},
+		Value:   params.DefaultProposerPolicy,
 	}
 
 	istSubGroupFlag = cli.Uint64Flag{
-		Name:  "ist-subgroup",
-		Usage: "governance subgroup size [default: 21]",
-		Value: params.DefaultSubGroupSize,
+		Name:    "ist-subgroup",
+		Usage:   "governance subgroup size [default: 21]",
+		Aliases: []string{"genesis.consensus.istanbul.subgroup"},
+		Value:   params.DefaultSubGroupSize,
 	}
 
 	cliqueEpochFlag = cli.Uint64Flag{
-		Name:  "clique-epoch",
-		Usage: "clique epoch",
-		Value: params.DefaultEpoch,
+		Name:    "clique-epoch",
+		Usage:   "clique epoch",
+		Aliases: []string{"genesis.consensus.clique.epoch"},
+		Value:   params.DefaultEpoch,
 	}
 
 	cliquePeriodFlag = cli.Uint64Flag{
-		Name:  "clique-period",
-		Usage: "clique period",
-		Value: params.DefaultPeriod,
+		Name:    "clique-period",
+		Usage:   "clique period",
+		Aliases: []string{"genesis.consensus.clique.period"},
+		Value:   params.DefaultPeriod,
 	}
 
 	istanbulCompatibleBlockNumberFlag = cli.Int64Flag{
-		Name:  "istanbul-compatible-blocknumber",
-		Usage: "istanbulCompatible blockNumber",
-		Value: 0,
+		Name:    "istanbul-compatible-blocknumber",
+		Usage:   "istanbulCompatible blockNumber",
+		Aliases: []string{"genesis.hardfork.istanbul-compatible-blocknumber"},
+		Value:   0,
 	}
 
 	londonCompatibleBlockNumberFlag = cli.Int64Flag{
-		Name:  "london-compatible-blocknumber",
-		Usage: "londonCompatible blockNumber",
-		Value: 0,
+		Name:    "london-compatible-blocknumber",
+		Usage:   "londonCompatible blockNumber",
+		Aliases: []string{"genesis.hardfork.london-compatible-blocknumber"},
+		Value:   0,
 	}
 
 	ethTxTypeCompatibleBlockNumberFlag = cli.Int64Flag{
-		Name:  "eth-tx-type-compatible-blocknumber",
-		Usage: "ethTxTypeCompatible blockNumber",
-		Value: 0,
+		Name:    "eth-tx-type-compatible-blocknumber",
+		Usage:   "ethTxTypeCompatible blockNumber",
+		Aliases: []string{"genesis.hardfork.eth-tx-type-compatible-blocknumber"},
+		Value:   0,
 	}
 
 	magmaCompatibleBlockNumberFlag = cli.Int64Flag{
-		Name:  "magma-compatible-blocknumber",
-		Usage: "magmaCompatible blockNumber",
-		Value: 0,
+		Name:    "magma-compatible-blocknumber",
+		Usage:   "magmaCompatible blockNumber",
+		Aliases: []string{"genesis.hardfork.magma-compatible-blocknumber"},
+		Value:   0,
 	}
 
 	koreCompatibleBlockNumberFlag = cli.Int64Flag{
-		Name:  "kore-compatible-blocknumber",
-		Usage: "koreCompatible blockNumber",
-		Value: 0,
+		Name:    "kore-compatible-blocknumber",
+		Usage:   "koreCompatible blockNumber",
+		Aliases: []string{"genesis.hardfork.kore-compatible-blocknumber"},
+		Value:   0,
 	}
 
 	kip103CompatibleBlockNumberFlag = cli.Int64Flag{
```

Comment:
I think aliases should be added for belows

Reply:
It doesn't matter, anyway modified for consistency